Source/Scene/Cesium3DTileset.js

```diff
@@ -1501,6 +1557,11 @@ define([
         // Number of commands added by each update above
         statistics.numberOfCommands = (commandList.length - numberOfInitialCommands);
 
+        if (addedCommandsLength > 0) {
+            // TODO : only do this if the tileset is purely point clouds.
```

Comment:
Is this TODO needed anymore?

Also just to make it clear that the processor only runs when it's enabled, the check could be
`if (tileset.pointCloudPostProcessorOptions.enabled && (addedCommandLength > 0))`